File: 11/AP/14/Soal56.java
public class Soal56
{
    public static void main(String[] args)
    {
        int[] list1 = {13, 11, 22, 13, 45};
        System.out.println(mystery(list1));

    }

    public static boolean mystery(int[] list1)
    {
        if (list1[0] == list1[list1.length - 1])
        {
            return true;
        }
        return false;
    }
}

File: 10/INFORMATIKA/C++/10-7/08/Soal56.cpp
#include <iostream>
#include <vector>
using namespace std;

string mystery(vector<int> &list1)
{
    if (list1[0] == list1[ list1.size()-1 ] )
    {
        return "True";
    }
    else
    {
        return "False";
    }
}

int main()
{
    vector<int> list1 = {13, 11, 22, 13, 45};
    cout << mystery(list1) << endl;
}